client/ayon_deadline/plugins/publish/harmony/submit_harmony_deadline.py

```diff
@@ -251,7 +252,8 @@ def get_job_info(self, job_info=None):
     def _unzip_scene_file(self, published_scene: Path) -> Path:
         """Unzip scene zip file to its directory.
 
-        Unzip scene file (if it is zip file) to its current directory and
+        Unzip scene file (if it is zip file) to work are if dir if available,
```

Comment:
```suggestion
        Unzip scene file (if it is zip file) to work area if dir if available,
```

It also states:
> if not to its current directory

Is that actually the case? I missed that part of the code?